monai/metrics/cumulative_average.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional
+
+import torch
+
+from monai.metrics.utils import do_metric_reduction
+from monai.utils import MetricReduction
+
+from .metric import CumulativeIterationMetric
+
+
+class CumulativeAverage(CumulativeIterationMetric):
```

Comment:
I think it's easier to follow if it inherits `Cumulative`, it doesn't have to be metrics...

Reply:
Thanks for your suggestion.
That's good idea, I updated in the latest commit.
Could you please help review it again?

Thanks.